jOOQ/pom.xml

```diff
@@ -115,24 +115,11 @@
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+        </dependency>
```

Comment:
Our tests are not open source, and this won't change. It's not an accident you didn't find a single test in our repository. See again: https://github.com/jOOQ/jOOQ/blob/main/CONTRIBUTING.md